crossdock/client/echo/thrift.go

```diff
@@ -42,7 +43,7 @@ func Thrift(t crossdock.T) {
 	fatals.NoError(dispatcher.Start(), "could not start Dispatcher")
 	defer dispatcher.Stop()
 
-	client := echoclient.New(dispatcher.Channel("yarpc-test"))
+	client := echoclient.New(dispatcher.Channel("yarpc-test"), thrift.DisableEnveloping)
```

Comment:
I would probably add the same TODO to each of these Crossdock tests that require enveloping disabled.